elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -346,6 +365,80 @@ public T getObject() {
     }
 
     @Override
+    public <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Optional<Expression> filterExpression
+    ) {
+        Object val = com.yahoo.elide.core.PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
I agree but I think this is out of scope for this PR.  I don't want to boil the ocean (PR is already pretty massive).